controllers/supervisor/supervisor.py

```diff
@@ -1103,7 +1116,7 @@ def is_in_opponent_penalty_area(x, y):
                             iy = sign * constants.ROBOT_FORMATION[constants.FORMATION_DEFAULT][id][1]
                             r = 1.5 * constants.ROBOT_SIZE[id]
                             # if any object is located within 1.5 * robot_size, the relocation is delayed
-                            if not self.any_object_nearby(ix, iy, r):
+                            if not self.any_object_nearby(ix, iy, r):                              
```

Comment:
This is probably useless.
```suggestion
                            if not self.any_object_nearby(ix, iy, r):
```